### DISCUSSION TITLE
ld: error: unable to find library -lndk_compat

### DISCUSSION THREAD

**Discussion by CMRomantic** [rustdesk/rustdesk#8955] [General]
error: linking with `/home/cmm/.cargo/bin/cargo-ndk` failed: exit status: 1
  ...
   "-nodefaultlibs" "-Wl,-rpath,$ORIGIN/../../../../../../.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/aarch64-linux-android/lib" "-Wl,--enable-new-dtags" "-Wl,-z,origin"
  = note: ld: error: unable to find library -lndk_compat
          clang-12: error: linker command failed with exit code 1 (use -v to see invocation)
          

warning: `rustdesk` (lib) generated 34 warnings
error: could not compile `rustdesk` (lib) due to 1 previous error; 34 warnings emitted
note: If the build failed due to a missing target, you can run this command:
note: 
note:     rustup target install aarch64-linux-android

ndk23c build Android platform has been prompted not to find this library

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/discussions/763, go to [discord](https://discord.gg/nDceKgxnkV) please, please search there first, many guys have asked similar question because you are not the most special though you always think you are. Usually nobody will help you with build or self-hosting here.
